### ISSUE TITLE
"Add" and "Save" buttons are not descriptive: A11y_.NET 9 template_ManageMeta_Name Role Value

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27461]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG4.1.2; #A11ySev3; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #NameRoleValue; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open. Swipe till Hamburger menu and double tap on it, Swipe till manage meta and double tap on it.
3. Swipe till "Save" and "Add buttons present under 'Categories' and 'Tags'.
4. Observe that "Add" and "Save" buttons are not descriptive.

### Actual Result:
The "Add" and "Save" buttons do not have descriptive labels. The voice-over currently announces them simply as 'Save' and 'Add' for the buttons located under 'Categories' and 'Tags'.

### Expected Result:
The "Add" and "Save" buttons should have more descriptive labels. The voice-over should announce them as 'Save Categories/ Save tags' and 'Add Categories / Add tags'.

### User Impact:
Screen reader users may face difficulty distinguishing which save and add buttons correspond to specific tasks, potentially causing confusion.

### Version with bug

9.0.30 SR3

### Is this a regression from previous behavior?

No, this is something new

### Affected platforms

iOS

### Relevant log output

```shell

```

**Comment by PoojaNamde** [comment]
Verified the bug in environment
Bug is fixed and does not repro for 'Save' and 'Add' for the buttons located under 'Categories' and 'Tags'. But still reproducing in Project Screen, Dashboard screen. Please find repro attachment for it.

https://github.com/user-attachments/assets/a01a8d6f-d16d-4680-a9b6-f700ac8bde34

**Comment by sheiksyedm** [comment]
"Add task" button and "Add project" button issue was resolved with this PR. 
https://github.com/dotnet/maui-samples/pull/650

**Comment by jsuarezruiz** [comment]
Fixed by https://github.com/dotnet/maui-samples/pull/650

**Comment by PoojaNamde** [comment]
@jsuarezruiz Could you please share the latest env to verify the issue.
